repsheet_backend/common.py

```diff
@@ -131,9 +157,14 @@ class BillVotingRecord(BaseModel):
     billNumber: str
     memberVote: Literal["yea", "nay", "abstain"]
     billBecameLaw: bool
+    billIsBudget: bool
     privateBillOfMember: bool
     membersPartyVote: PartyVotes
     issues: BillIssues
+    memberInGovernment: bool
+    memberInOpposition: bool
+    memberInSupplyAndConfidence: bool
+    parliamentYeaPercentage: float
```

Comment:
I've been putting percentages as proper %age strings, e.g. "81%", since I have an unvalidated assumption that this will be easier for the AI to understand (assumption of relative prevalence in training data).